source/extensions/filters/http/fault/config.cc

```diff
@@ -21,6 +21,17 @@ Http::FilterFactoryCb FaultFilterFactory::createFilterFactoryFromProtoTyped(
   };
 }
 
+Http::FilterFactoryCb FaultFilterFactory::createFilterFactoryFromProtoWithServerContextTyped(
+    const envoy::extensions::filters::http::fault::v3::HTTPFault& config,
+    const std::string& stats_prefix, Server::Configuration::ServerFactoryContext& server_context) {
+  FaultFilterConfigSharedPtr filter_config(
```

Comment:
nit: I know it is from old code. can we use `make_shared` instead of `new` for modern c++ code. https://abseil.io/tips/126